packages/web-app-files/src/components/SideBar/Shares/Collaborators/ListItem.vue

```diff
@@ -193,11 +214,13 @@ export default defineComponent({
       if (!this.share.collaborator.additionalInfo) {
         return
       }
-      return ` (${this.share.collaborator.additionalInfo})`
+      return this.share.collaborator.additionalInfo
```

Comment:
You could skip the early return above, no? This line here is sufficient for the computed prop.